samples/pollion_dash/data_preparation.R

```diff
@@ -0,0 +1,178 @@
+working_dir<-getwd()
+source(paste(working_dir, "/functions.r",sep="",collapse=NULL))
+
+
+##create vector of packages to either install or load
+packages_to_install <- c("ggplot2","shiny","reshape","scales","grid","gridExtra",
+                         "rjson","tm","wordcloud","RColorBrewer","psych")
+packages_to_install
+
+#install.packages(packages_to_install)
+##use lapply to be able to use library() despite quatation marks
+lapply(packages_to_install, library, character.only=T)
+
+############################ data manipulation
+#get and set working directory
+working_dir<-getwd()
+working_dir
+
+startpoll_json_file<-"/Users/Thomas/Documents/app_pollion/app_pasta_neu/data/pasta_data2.json"
```

Comment:
please do not use paths which are working only for yourself, the startpoll_json_file should be accessible for other programmers as well, so always try to use a relative path and add a prerequisites paragraph as comment on top of a script